js/ui/grid_core/ui.grid_core.validating.js

```diff
@@ -592,8 +752,23 @@ module.exports = {
                         callBase.call(this, $cell);
                     }
                 },
+
                 getEditDataByKey: function(key) {
                     return this._editData[getIndexByKey(key, this._editData)];
+                },
+
+                getEditData: function() {
```

Comment:
I can't find any usage of this method.

Reply:
It' is not used anymore. I forgot to delete it.